draft-ietf-quic-qpack.md

```diff
@@ -265,6 +265,52 @@ might not actually become blocked on every stream which risks becoming blocked.
 If the decoder encounters more blocked streams than it promised to support, it
 SHOULD treat this as a stream error of type HTTP_QPACK_DECOMPRESSION_FAILED.
 
+### State Synchronization
+
+The decoder stream signals key events at the decoder that permit the encoder to
+track the decoder's state.  These events are:
+
+- Successful processing of a header block
+- Abandonment of a stream which might have remaining header blocks
+- Receipt of new dynamic table entries
+
+Regardless of whether a header block contained blocking references, the
+knowledge that it was processed successfully permits the encoder to avoid
+evicting entries while references remain outstanding; see {{blocked-eviction}}.
+When a stream is reset or abandoned, the indication that these header blocks
+will never be processed serves a similar function.
+
+For the encoder to identify which dynamic table entries can be safely used
+without a stream becoming blocked, the encoder tracks the Largest Known Received
+absolute index in the decoder's state.
```

Comment:
"in the decoder's state" sounds kind of weird. Maybe "the encoder tracks the absolute index of the decoder's Largest Known Received entry.